packages/customWidgets/calendar-web/src/components/CalendarContainer.ts

```diff
@@ -151,12 +153,27 @@ export default class CalendarContainer extends Component<Container.CalendarConta
         return new Date();
     };
 
+    private getDisabledTill = async (mxObject: MxObject): Promise<Date> => {
+        if (mxObject && this.props.disabledTill) {
+            let disabledTill;
+            try {
+                disabledTill = await this.extractAttributeValue<number>(mxObject, this.props.disabledTill);
+            } catch (error) {
+                window.mx.ui.error(`Unable to fetch start date attribute value: ${error.message}`);
```

Comment:
"start date" --> "disabledTill date"

Reply:
Misread, i changed the attribute to disabledTillDate